md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessLogAPI.java

```diff
@@ -93,14 +81,10 @@ RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage
     public
     @ResponseBody
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)